en/concept/core_components.md

```diff
@@ -0,0 +1,91 @@
+| [Vision][Vision] (State Estimation)                                                |                                      |                                                  |
+| [Boards (NuttX)][Boards (NuttX)]                                                   | [Pixhawk standard][Pixhawk standard] |                                                  |
+| IMU (Driver)                                                                       |                                      |                                                  |
+| Distance Sensor (Driver)                                                           |                                      |                                                  |
+| Failure Injection (?)                                                              |                                      |                                                  |
+| RTK GPS (Driver)                                                                   | Heading estimate from dual RTK GPS   | [Alex Klimaj](https://github.com/AlexKlimaj)     |
+| [Documentation][Documentation]                                                     | PX4, MAVLink, QGC, MAVSDK            | [Hamish Willee](https://github.com/hamishwillee) |
+| Battery (Library)                                                                  |                                      |                                                  |
+| Flight Task (Library)                                                              |                                      |                                                  |
+| Matrix (Library)                                                                   |                                      |                                                  |
+| Autotuning (Library)                                                               |                                      |                                                  |
+| [Github Actions (Continuous Integration)][Github Actions (Continuous Integration)] |                                      |                                                  |
+| [Jenkins (Continuous Integration)][Jenkins (Continuous Integration)]               |                                      |                                                  |
+| Linux (Toolchain)                                                                  |                                      |                                                  |
+| Mac OS (Toolchain)                                                                 |                                      |                                                  |
+| [Windows (Toolchain)][Windows (Toolchain)]                                         | WSL2                                 |                                                  |
```

Comment:
The windows toolchain is just WLS2 IMO. Cygwin and VMs are defunct or community supported. So I would remove the WSL2 subcomponent.